pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -272,13 +286,14 @@ func (rc *reconciler) reconcile() {
 			}
 		}
 	}
+}
 
-	// Ensure devices that should be detached/unmounted are detached/unmounted.
```

Comment:
do you meant to comment about this function ?

Reply:
I moved this to the caller.